roles/openshift_certificate_expiry/README.md

```diff
@@ -104,35 +139,71 @@ $ ansible-playbook -v -i HOSTS /usr/share/ansible/openshift-ansible/playbooks/ce
 
 > [View This Playbook](../../playbooks/certificate_expiry/easy-mode.yaml)
 
-***
+### Easy mode and upload reports to masters
+
+This example builds on top of [easy-mode.yaml](#easy-mode) and additionally
+uploads a copy of the generated reports to the masters, with a timestamp in the
+file names.
+
+This is specially useful when the playbook runs from within a container, because
+the reports are generated inside the container and we need a way to access them.
+Uploading a copy of the reports to the masters is one way to make it easy to
+access the reports. Alternatively you can use the
+[role variables](#role-variables) that control the path of the generated reports
+to point to a container volume.
 
-Default behavior:
+With the container use case in mind, this playbook allows control over some
+options via environment variables:
+
+ - `CERT_EXPIRY_WARN_DAYS`: sets `openshift_certificate_expiry_warning_days`,
```

Comment:
This customization hook is great. I comment about it though because it diverges from the roles default warning window. The role parameters document `openshift_certificate_expiry_warning_days` as having a default value of `30`. Would you consider changing the defaults from `45` to match the `30` for the role? I'm open to hearing why you might not want to as well.